typescript-api/scripts/runtime-upgrade.ts

```diff
@@ -41,6 +41,10 @@ function hackXcmV5Support() {
     // For dancelight, replace "Lookup77" with "StagingXcmV5Junction"
     const dancelightFilePath = "src/dancelight/interfaces/types-lookup.ts";
     hackTypeReplacement(dancelightFilePath, "Lookup76", "StagingXcmV5Junction", 8);
+
+    // For starlight, replace "Lookup77" with "StagingXcmV5Junction"
+    const starlightFilePath = "src/starlight/interfaces/types-lookup.ts";
+    hackTypeReplacement(starlightFilePath, "Lookup76", "StagingXcmV5Junction", 8);
```

Comment:
Hopefully we can remove this hack soon, when this is fixed: https://github.com/polkadot-js/api/issues/6087